tests/test_rdf.py

```diff
@@ -86,3 +86,64 @@ def compare_gromacs_xvg(ref_filename, r, g_r):
     g_r0 = data[:, 1]
     eq(r, r0, decimal=2)
     eq(g_r, g_r0, decimal=2)
+
+@pytest.mark.parametrize('periodic, opt', [(True, True), (True, False), (False, True), (False, False)])
+def test_rdf_t(get_fn, periodic, opt):
+    traj = md.load(get_fn('tip3p_300K_1ATM.xtc'), top=get_fn('tip3p_300K_1ATM.pdb'))
+    pairs = traj.top.select_pairs('name O', 'name O')
+    times = list()
+    for j in range(2):
+        for i in range(4):
+            times.append([j*4+i, j*4])
+
+    r, g_r_t = md.compute_rdf_t(traj, pairs, times, r_range=(0, 1), periodic=periodic, opt=opt)
+
+    assert g_r_t.shape == (8, 200)
+
+@pytest.mark.parametrize('time_diff', [0, 2, 4])
+def test_rdf_t_norm(get_fn, time_diff):
+    # Check if the tail of g(r,t) at t=0 and t=time_diff is normalized to ~1.0
+    traj = md.load(get_fn('tip3p_300K_1ATM.xtc'), top=get_fn('tip3p_300K_1ATM.pdb'))
+
+    times = list()
+    for j in range(100):
+        if (j + time_diff) > traj.n_frames:
+            continue
+        times.append([j, j+time_diff])
+
+    pairs = traj.top.select_pairs('name O', 'name O')
+    r_t, rdf_O_O = mdtraj.geometry.rdf.compute_rdf_t(traj, pairs, times)
+    assert eq(np.ones(20), np.mean(rdf_O_O, axis=0)[-20:], decimal=1)
+
+    pairs = traj.top.select_pairs('name O', "name =~ 'H.*'")
+    r_t, rdf_O_H = mdtraj.geometry.rdf.compute_rdf_t(traj, pairs, times)
+    assert eq(np.ones(20), np.mean(rdf_O_H, axis=0)[-20:], decimal=1)
+
+@pytest.mark.parametrize('periodic, opt', [(True, True), (True, False), (False, True), (False, False)])
+def test_compare_rdf_t_at_0(get_fn, periodic, opt):
+    # Compute g(r, t) at t = 0 and compare to g(r)
+    frames = 2
+    traj = md.load(get_fn('tip3p_300K_1ATM.xtc'), top=get_fn('tip3p_300K_1ATM.pdb'))
+    pairs = traj.top.select_pairs('name O', 'name O')
+    times = list()
+    for j in range(frames):
+        times.append([j, j])
+
+    r_t, g_r_t = md.compute_rdf_t(traj, pairs, times, self_correlation=False, periodic=periodic, opt=opt)
+    mean_g_r_t = np.mean(g_r_t, axis=0)
+    r, g_r = md.compute_rdf(traj[:frames], pairs, periodic=periodic, opt=periodic)
+
+    assert eq(r_t, r)
+    assert eq(mean_g_r_t, g_r, decimal=1)
+
+@pytest.mark.skip('Binning does not match up with gromacs currently.')
+def test_compare_gromacs_rdf_t(get_fn):
```

Comment:
Which if these tests cases are triclinic? That seems like an important case to test thoroughly.

Reply:
I added a triclinic system for testing as a start.